docs/templates.md

```diff
@@ -99,6 +99,31 @@ $pagination->setCustomParameters([
 ]);
 ```
 
+If you want to use the template with the same parameters everywhere, create your own
+template and let extend it from the provided template.
+
+Here's an example how to achieve that
```

Comment:
Here's an example **of** how to achieve that

Reply:
Thanks. I have fixed it